### PR TITLE
feat(core): allow for options in sendUserActivation

### PR DESCRIPTION
**Pull request by shnaaz**
…ough to sendUserActivation

Allow for extra options to be passed through to sendUserActivation when sending the activation and to use license service.
This will be useful for webex web app in user activation flow to validate user.

Fixes #[INSERT LINK TO ISSUE NUMBER]

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by mwegman — APPROVED** [review]

File: packages/node_modules/@webex/webex-core/src/lib/services/services.js [hunk 1]
```diff
@@ -363,15 +369,18 @@ const Services = WebexPlugin.extend({
...
   /**
    * Send a request to activate a user using a client token.
    *
    * @param {SendUserActivationPTO} - The Parameter transfer object.
-   * @returns {AtlasDTO} - The DTO returned from the **Atlas** service.
+   * @returns {LicenseDTO} - The DTO returned from the **License** service.
    */
-  sendUserActivation({email, reqId, token}) {
+  sendUserActivation({
+    email, reqId, token, activationOptions
```
Comment (taymoork2) [inline packages/node_modules/@webex/webex-core/src/lib/services/services.js:382]: new line params for readability
```suggestion
    email,
    reqId,
    token,
    activationOptions
```

File: packages/node_modules/@webex/webex-core/src/lib/services/services.js [hunk 2]
```diff
@@ -321,7 +325,9 @@ const Services = WebexPlugin.extend({
           exists: true,
           details: 'user exists and is activated'
         },
-        this.sendUserActivation({email, reqId, token})
+        this.sendUserActivation({
+          email, reqId, token, activationOptions
```
Comment (taymoork2) [inline packages/node_modules/@webex/webex-core/src/lib/services/services.js:329]: new line params for readability
```suggestion
          email,
          reqId,
          token,
          activationOptions
```

File: packages/node_modules/@webex/webex-core/src/lib/services/services.js [hunk 3]
```diff
@@ -240,7 +241,9 @@ const Services = WebexPlugin.extend({
    * @param {ValidateUserPTO} - The parameter transfer object.
    * @returns {ValidateUserRTO} - The return transfer object.
    */
-  validateUser({email, reqId = 'WEBCLIENT', forceRefresh = false}) {
+  validateUser({
+    email, reqId = 'WEBCLIENT', forceRefresh = false, activationOptions = {}
```
Comment (taymoork2) [inline packages/node_modules/@webex/webex-core/src/lib/services/services.js:245]: new line params for readability
```suggestion
    email,
    reqId = 'WEBCLIENT',
    forceRefresh = false,
    activationOptions = {}
```

File: packages/node_modules/@webex/webex-core/src/lib/services/services.js [hunk 4]
```diff
@@ -394,14 +403,14 @@ const Services = WebexPlugin.extend({
             'x-prelogin-userid': undefined
           },
           body: {
-            email, reqId, countryCode, timeZone: timezone
+            email, reqId, countryCode, timeZone: timezone, ...activationOptions
```
Comment (taymoork2) [inline packages/node_modules/@webex/webex-core/src/lib/services/services.js:406]: new params for readability
```suggestion
            email,
            reqId,
            countryCode,
            timeZone: timezone,
            ...activationOptions
```

**Review by taymoork2 — CHANGES_REQUESTED** [review]
1. Amend the changes requested into the 1st commit
2. Rewrite first commit's scope to `feat(webex-core)`
3. Rewrite second commit's scope to `test(webex-core)`

**Comment by lalli-flores** [comment]
@shnaaz, please make sure to keep your commits cleanly separate changes as stated in the contributing guide, as they translate into the changelog. If you're addressing feedback, do not create another commit, you need to rebase your existing commit

**Review by taymoork2 — APPROVED** [review]
LGTM

**Pull request merged**